fastlane/Fastfile

```diff
@@ -0,0 +1,58 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:android)
+
+platform :android do
+  gradle(
+    task: "assemble",
+    build_type: "Release",
+    print_command: false,
+    properties: {
+        "android.injected.signing.store.password" => "store_password",
+        "android.injected.signing.key.alias" => "key_alias",
+        "android.injected.signing.key.password" => "key_password",
+    }
+  )
+
+  desc "Deploy a new version to Slack Channel"
+  lane :deploy do
+    gradle(task: "clean assembleRelease")
+    slack_build()
+  end
+
+  desc "Upload new version to Slack Channel"
+  private_lane :slack_build do |options|
+    file_path = lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]
+    file_name = file_path.gsub(/\/.*\//,"")
+    token = ENV["SLACK_URL"]
```

Comment:
the token is different from the `webhook url`. please generate a token which will authorize your upload to slack

Reply:
Noted